tools/generators/package/index.js

```
@@ -1,28 +1,79 @@
+    tmpl: "",
+    name: normalized.fileName,
+    className: normalized.className,
+    propertyName: normalized.propertyName,
+    constantName: normalized.constantName,
+=======*/
   const variants = names(schema.name);
   const normalized = variants.fileName;
   const preset = schema.preset ?? "base";
   const templateDirectory = joinPathFragments(__dirname, "files", preset);
   const projectRoot = joinPathFragments("packages", normalized);
+  const templateDirectory = resolveTemplateDirectory(schema.preset);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'templateDirectory'. Consider to delete it or rename it.

'templateDirectory' is defined here:


(lint/suspicious/noRedeclare)